content/applications/finance/fiscal_localizations/ecuador.rst

```diff
@@ -0,0 +1,570 @@
+
+These types of electronic documents can be created from the :guilabel:`Purchase Order` or manually
+from the :guilabel:`Vendor Bills` form view. It must contain the following data:
+
+- :guilabel:`Vendor`: type the vendor's information.
+- :guilabel:`Journal`: select the journal for the :guilabel:`Purchase Liquidation` with the correct
+  printer point.
+- :guilabel:`Document Type`: this is the document type `(03) Purchase Liquidation`
+- :guilabel:`Document number`: type the document number (sequence), you will only have to do this
+  once, then the sequence will be automatically assigned for the next documents.
+- :guilabel:`Payment Method (SRI)`: select how the invoice is going to be paid.
+- :guilabel:`Products`: specify the product with the correct taxes.
+
+Once you review the information you can validate the :guilabel:`Purchase Liquidation`.
+
+.. image:: ecuador/purchase-liquidation.png
+   :align: center
+   :alt: Purchase liquidation for Ecuador.
+
+Purchase withholding
+~~~~~~~~~~~~~~~~~~~~
+
+The :guilabel:`Purchase withholding` is an electronic document that, when validated, is sent to SRI.
+
+It is necessary to have an invoice in a validated state in order to register a :guilabel:`Purchase
+withholding`. On the invoice, there is a button named :guilabel:`Add Withhold`, click on this button
+to be directed to the :guilabel:`Withholding` form, then complete the following information:
+
+- :guilabel:`Document number`: type the document number (sequence), you will only have to do this
+  once, then the sequence will be automatically assigned for the next documents.
+- :guilabel:`Withhold lines`: The taxes appear automatically according to the configuration of
+  products and vendors, you should review if the taxes and tax support are correct, and, if it is not
+  correct, you can edit and select the correct taxes and tax support.
+
+Once you review the information you can validate the :guilabel:`Withholding`.
+
+.. image:: ecuador/purchase-withhold.png
+   :align: center
+   :alt: Purchase withhold for Ecuador.
+
+.. note::
+   You can't change the tax support for one that was not included in the configuration of the taxes
+   used on the :guilabel:`Vendor Bill`. To do so, go to the tax applied on the :guilabel:`Vendor
+   Bill` and change the :guilabel:`Tax Support` there.
+
+A withholding tax can be divided into two or more lines, this will depend on whether two or more
+withholdings percentages apply.
+
+.. example::
+   The system suggests a VAT withholding of 30% with tax support 01, you can add your VAT withholding
+   of 70% in a new line with the same tax support, the system will allow you as long as the total of
+   the bases matches the total from the :guilabel:`Vendor Bill`.
+
+Financial Reports
+=================
+
+In Ecuador, there are fiscal reports that the company presents to SRI. In Odoo, we have two of the
+main financial reports used by companies. These are the reports 103 and 104.
```

Comment:
I would add the path to both reports in this section, and only one screenshot to illustrate where to find them would be sufficient.  
![image](https://user-images.githubusercontent.com/85342080/219618281-8987ef98-9030-4bed-947d-25bac932402a.png)
Also, could you please check the path to the reports? the tax report appears in the Statement Reports on my runbot
![image](https://user-images.githubusercontent.com/85342080/219618500-5e926555-8fc0-4208-85b8-b967bfbda6cb.png)